packages/ast-position/test/ast-position-spec.js

```diff
@@ -0,0 +1,69 @@
+const { getAstNodeInPosition } = require("../lib/api");
+const { buildAst } = require("@xml-tools/ast");
+const { expect } = require("chai");
+const { TextDocument } = require("vscode-languageserver");
+const { parse } = require("@xml-tools/parser");
+
+describe("AST Position visitor", () => {
```

Comment:
Add negative tests that return undefined:
* Not inside element (before first `<`, after last `>`, between elements)
* After tag name but before the first attribute
* Between attributes
* On the `=` of the attribute (if you have a problem reproducing it add some whitespaces)